StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OpenSecureWebViewMessageHandler.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  OpenSecureWebViewMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/14/24.
+//
+
+import Foundation
+
+// Indicates to open the provided URL in a secure webview. 
+class OpenSecureWebViewMessageHandler: ScriptMessageHandler<OpenSecureWebViewMessageHandler.Message> {
```

Comment:
🚫 **Blocker:** Naming
```suggestion
// Indicates to open the provided URL in an `ASWebAuthenticationSession`. 
class OpenAuthenticatedWebViewMessageHandler: ScriptMessageHandler<OpenSecureWebViewMessageHandler.Message> {
```

Rationale: https://docs.google.com/document/d/1Mo89Fca4T_qu1ayb-kA6nFLGB7sdn0JxItj-wJBvzT0/edit?pli=1#bookmark=id.wh254hfy3uy0

We can leave the string message name as-is for now, but I'm going to propose we rename this to `openAuthenticatedWebView`.